src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ page import="com.skytask.model.Product" %>
+<%@ page import="java.util.List" %>
+
+<%
+List<Product> products = (List<Product>) request.getAttribute("products");
+%>
+
+<table>
+
+<tr>
+<td>Name</td>
+<td>Description</td>
+<td>Price</td>
+<td>Stock</td>
+</tr>
+
+<% for(Product p : products){ %>
```

Comment:
it looks cleaner using jstl: 
https://www.tutorialspoint.com/jsp/jstl_core_foreach_tag.htm